{{cookiecutter.project_slug}}/package.json

```diff
@@ -8,46 +8,67 @@
     "postinstall": "npx jetify && rndebugger-open",
     "lint": "eslint .",
     "start": "react-native start",
-    "test": "jest"
+    "test": "jest",
+    "web": "react-app-rewired start",
+    "web-build": "react-app-rewired build && mkdir -p backend/web_build && cp -R build/* backend/web_build"
   },
   "jest": {
     "preset": "react-native"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.10",
+    "@react-native-community/slider": "^3.0.3",
+    "@react-navigation/drawer": "^5.12.3",
+    "@react-navigation/native": "^5.9.2",
+    "@react-navigation/stack": "^5.14.2",
     "axios": "0.19.0",
     "lodash": "4.17.20",
     "moment": "2.27.0",
     "react": "16.11.0",
+    "react-dom": "^17.0.1",
     "react-native": "0.62.0",
+    "react-native-datepicker": "^1.7.2",
     "react-native-debugger-open": "0.3.24",
-    "react-native-gesture-handler": "1.6.1",
+    "react-native-dotenv": "^0.2.0",
+    "react-native-elements": "^2.3.2",
+    "react-native-fix-image": "^2.1.0",
+    "react-native-gesture-handler": "^1.10.1",
+    "react-native-image-picker": "^1.1.0",
+    "react-native-keyboard-aware-scroll-view": "0.9.1",
     "react-native-maps": "^0.27.1",
-    "react-native-reanimated": "1.9.0",
-    "react-native-screens": "1.0.0-alpha.22",
+    "react-native-reanimated": "^1.13.2",
+    "react-native-safe-area-context": "^3.1.9",
+    "react-native-screens": "^2.17.1",
     "react-native-svg": "9.13.6",
     "react-native-vector-icons": "^6.6.0",
+    "react-native-web": "^0.14.13",
     "react-navigation": "^4.4.0",
+    "react-navigation-drawer": "^2.3.4",
     "react-navigation-stack": "1.10.3",
```

Comment:
Those should be removed given that you're upgrading to react-navigation v5

Reply:
should it be there for backward compatible ?

Reply:
Me and @tan-duong discussed this in Slack. This has to stay for now because some of our modules still depend on it.